src/Layout/Footer/Footer.js

```diff
@@ -18,6 +30,12 @@ const Footer = () => {
         currentTime={current.position}
       />
       <FooterWrapper className="footer">
+        <FontAwesomeIcon
+          icon="forward"
+          title="Skip current song"
+          className="skip-button"
```

Comment:
I know we're currently using CSS classes to style Font Awesome icons, but maybe it would be great to start using `styled-components` for that.

Something like this:
```js
import React from 'react';
import styled from 'styled-components';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const StyledIcon = styled(FontAwesomeIcon).`
  color: red;
`;

const Icon = () => (
   <StyledIcon
      icon={['fas', 'balance-scale']}
      fixedWidth
      size="5x"
   />
 );

export default Icon;
```

Reply:
I'd keep it like this for the PR and address it in a standalone PR which would update all current icons if we end up going for it

Reply:
Leaving for the next PR then.